### PR TITLE
Rename yosys, yosys-plugins and vtr packages

### PR DESCRIPTION
**Pull request by kowalewskijan**
Renaming packages was discussed with @mithro 

Additionally I changed **readline <8** to **readline** in vtr package due to conflicts when I tried to install the newest packages.

### CODE REVIEW

File: symbiflow-xray-yosys/meta.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v','') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
 
 package:
-  name: yosys
+  name: symbiflow-xray-yosys
```
Comment (mithro) [inline symbiflow-xray-yosys/meta.yaml]: I don't think the SymbiFlow is X-Ray specific?
Probably should just be `symbiflow-yosys`?

File: symbiflow-xray-yosys-plugins/meta.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v','') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
 
 package:
-  name: yosys-plugins
+  name: symbiflow-xray-yosys-plugins
```
Comment (mithro) [inline symbiflow-xray-yosys-plugins/meta.yaml]: `symbiflow-yosys-plugins`?

File: symbiflow-xray-vtr/meta.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v','') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
 
 package:
-  name: vtr
+  name: symbiflow-xray-vtr
```
Comment (mithro) [inline symbiflow-xray-vtr/meta.yaml]: `symbiflow-vtr`?

File: symbiflow-xray-yosys/meta.yaml (hunk 1 above)
Comment (kowalewskijan) [reply symbiflow-xray-yosys/meta.yaml]: Changed to `symbiflow-yosys`

File: symbiflow-xray-vtr/meta.yaml (hunk 3 above)
Comment (kowalewskijan) [reply symbiflow-xray-vtr/meta.yaml]: Changed to `symbiflow-vpr`

File: symbiflow-xray-yosys-plugins/meta.yaml (hunk 2 above)
Comment (kowalewskijan) [reply symbiflow-xray-yosys-plugins/meta.yaml]: Changed to `symbiflow-yosys-plugins`

**Comment by PiotrZierhoffer** [comment]
I reimplemented these changes in #101, so I'll close this one.

I did that to combine changes of naming in one PR

**Pull request closed without merging**